packages/strapi-plugin-content-type-builder/admin/src/containers/ModelPage/index.js

```diff
@@ -198,7 +192,7 @@ export class ModelPage extends React.Component {
       ? this.getModelAttributesLength() > 0
       : !isEqual(
           modifiedData[this.getModelName()],
-          initialData[this.getModelName()],
+          initialData[this.getModelName()]
```

Comment:
Can you configure your editor so when your format the file onSave it also adds the trailing comas?